src/com/alevel/cw7/models/Player.java

```diff
@@ -0,0 +1,153 @@
+package com.alevel.cw7.models;
+
+import java.util.Arrays;
+
+public class Player {
+    private final String name;
+    private final Type type;
+    private int age;
+    private double satiety;
+    private double money;
+    private double purity;
+    private double happiness;
+    private Present[] presents;
+
+    private boolean sick;
+
+    public Player(String name, Type type) {
+        this.name = name;
+        this.type = type;
+        this.age = 1;
+        this.satiety = 100;
+        this.purity = 100;
+        this.happiness = 50;
+        presents = new Present[0];
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public double getSatiety() {
+        return satiety;
+    }
+
+    public double getMoney() {
+        return money;
+    }
+
+    public double getPurity() {
+        return purity;
+    }
+
+    public double getHappiness() {
+        return happiness;
+    }
+
+    public Present[] getPresents() {
+        return presents;
+    }
+
+    public void setHappiness(double happiness) {
+        this.happiness = happiness;
+    }
+
+    public void setSick(boolean sick) {
+        this.sick = sick;
+    }
+
+
+    public void setSatiety(double satiety) {
+        this.satiety = satiety;
+    }
+
+    public void setPresents(Present[] presents) {
+        this.presents = presents;
+    }
+
+    public void setPurity(double purity) {
+        this.purity = purity;
+    }
+
+    public int isUnPurity() {
+        return (purity < 50) ? 3 : 1;
+    }
+
+    public double isHungry() {
+        return (satiety < 50.0) ? 1.5 : 1.0;
+    }
+
+    public boolean isSick() {
+        return sick;
+    }
```

Comment:
I would do in a similar way to other methods for checking the status:
 return sick ? 2 : 1;